Cargo.toml

```diff
@@ -47,6 +47,7 @@ exr = { version = "1.5.0", optional = true }
 gif = { version = "0.13", optional = true }
 image-webp = { version = "0.2.0", optional = true }
 mp4parse = { version = "0.17.0", optional = true }
+pcx = { version = "0.2.3", optional = true }
```

Comment:
This dependency is small, and licensed WTFPL. I got the test PCX images from their repository ( https://github.com/kryptan/pcx/tree/master/test-data 

The test suite in this project is rendering them as they appear in that repository, but it's worth noting that there are bugs in the decoder's implementation, particularly visible in cga_bw and cga_tst1.